### PR TITLE
bump to hippo v0.19.0

### PR DESCRIPTION
**Pull request by bacongobbler**
To be merged after https://github.com/fermyon/spin/pull/678 is merged and a new release of spin becomes available.

### CODE REVIEW

File: azure/terraform/main.tf [hunk 1]
```diff
@@ -21,8 +21,8 @@ locals {
   spin_version  = "v0.4.0"
```
Comment (vdice) [inline azure/terraform/main.tf:21]: Shall we bump spin as well to `"v0.5.0"`?

linux_amd64 checksum is `"c19f247730db0c54957a15b4402348be01742425a461431241213f15b8971987"`; or grab from https://github.com/fermyon/spin/releases/tag/v0.5.0

**Review by vdice — COMMENTED** [review]
With the v0.5.0 release of Spin, we're ready to move forward with this one!  🎉 

We'll want to give it a quick test before merging...

**Comment by vdice** [comment]
@bacongobbler my testing is looking good when using spin v0.5.0 with hippo built from HEAD (particularly to get the fix in https://github.com/deislabs/hippo/issues/1167) -- can we cut a new release of hippo and then update this PR?

**Comment by bacongobbler** [comment]
Yes! I can find some time this week to cut a new release.

**Comment by vdice** [comment]
Superseded by https://github.com/fermyon/installer/pull/105.  We can follow-up with the next hippo release to get a fix for when the postgresql backend is used (ref https://github.com/deislabs/hippo/issues/1167)

**Pull request closed without merging**